api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -159,17 +159,32 @@ type VolumeReplicationGroupSpec struct {
 	KubeObjectProtection *KubeObjectProtectionSpec `json:"kubeObjectProtection,omitempty"`
 }
 
+const ReservedBackupName = "use-backup-not-restore"
+
+type RecipeSpec struct {
+	//+optional
+	Name *string `json:"name,omitempty"`
+
+	//+optional
+	Workflow *WorkflowSpec `json:"workflow,omitempty"`
+}
+
+type WorkflowSpec struct {
+	//+optional
+	CaptureName *string `json:"captureName,omitempty"`
+
+	//+optional
+	RecoverName *string `json:"recoverName,omitempty"`
```

Comment:
(nit) I suggest `Workflow` be renamed `WorkflowNames` with fields name `Capture` and `Recover` OR the sub-struct be removed and replaced with a `CaptureWorkflowName` and `RecoverWorkflowName`